utils/keywords.py

```diff
@@ -0,0 +1,53 @@
+import pke 
+def search_keywords(text):
```

Comment:
**nitpick:** We could use typing, I like it more than docs, because in IDE it suggest you what you can expect as return from function.
```suggestion
def search_keywords(text) -> list[str]:
```

Reply:
even more we could add typing to arguments, like: `text: str`